tasks/61_spinal-case/spinal-case.js

```diff
@@ -0,0 +1,23 @@
+function spinalCase(str) {
+  const separators = [' ', '_'];
+  const arr = str.split('');
```

Comment:
You're changing every value of array so it looks like it is a case for `.map()` method.